README.md

```diff
@@ -17,10 +17,9 @@ If you want to use OpenBLAS, install it in any directory:
   make
 ```
```

Comment:
If you want you can leave out the middle sentence and just have "The blas/cgo package provides bindings to external BLAS libraries. More information about using cgo can be found at [link to http://golang.org/cmd/cgo/] "

Two sentences aren't a lot of cost, and it'll point people in the right direction if they're confused.